### PR TITLE
Added English README

### PR DESCRIPTION
**Pull request by jaschadub**

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# Command Line Tool Introduction
+Welcome to Tencent Cloud Command Line Tool (TCCLI), a unified tool for managing Tencent cloud resources. With Tencent Cloud Command Line Tools, you can quickly and easily call the Tencent Cloud API to manage your Tencent cloud resources. You can also automate and script based on Tencent Cloud's command-line tools, which can be combined and reused in a variety of ways.
+# Install TCCLI
+1. Install the Python environment and Pip tools. Before installing the command line tools, make sure your system has the Python environment and Pip tools installed. **Note that the python version must be 2.7 and above**, please refer to [python homepage](https://www.python.org/) and [pip homepage](https://pypi.org/project/pip/).
+2. TCCLI relies on the TencentCloudApi Python SDK. **If the version number of the TencentCloudApi Python SDK is smaller than the TCCLI version number to be installed, the TencentCloudApi Python SDK** will be automatically upgraded when TCCLI is installed.
```
Comment (zqfan) [inline README.en.md]: the whole sentence should be emphasized

File: README.en.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+3. Install TCCLI and execute the following command:
+```bash
+Pip install tccli
```
Comment (zqfan) [inline README.en.md]: Pip -> pip

File: README.en.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
+# Command Line Tool Introduction
+Welcome to Tencent Cloud Command Line Tool (TCCLI), a unified tool for managing Tencent cloud resources. With Tencent Cloud Command Line Tools, you can quickly and easily call the Tencent Cloud API to manage your Tencent cloud resources. You can also automate and script based on Tencent Cloud's command-line tools, which can be combined and reused in a variety of ways.
+# Install TCCLI
+1. Install the Python environment and Pip tools. Before installing the command line tools, make sure your system has the Python environment and Pip tools installed. **Note that the python version must be 2.7 and above**, please refer to [python homepage](https://www.python.org/) and [pip homepage](https://pypi.org/project/pip/).
+2. TCCLI relies on the TencentCloudApi Python SDK. **If the version number of the TencentCloudApi Python SDK is smaller than the TCCLI version number to be installed, the TencentCloudApi Python SDK** will be automatically upgraded when TCCLI is installed.
+3. Install TCCLI and execute the following command:
```
Comment (zqfan) [inline README.en.md]: Executing the following command to install TCCLI.

File: README.en.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+```bash
+Pip install tccli
+```
+4. After the installation is complete, execute tccli version to check whether the installation is successful.
```
Comment (zqfan) [inline README.en.md]: execute tccli version to -> execute `tccli version` to

File: README.en.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+5. If your environment is a Linux environment, you can enable auto-completion with the following command:
+```bash
+Complete -C 'tccli_completer' tccli
```
Comment (zqfan) [inline README.en.md]: Complete -> complete

File: README.en.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+secretId: Cloud API key SecretId.
+secretIKey: Cloud API key SecretKey.
+Region: Cloud product area, please go to the corresponding product page to get the available region.
+Output: Optional parameter, request return packet output format, support [json table text] three formats, the default is json.
+For more information, please see tccli configure help.
+
+2. Command line mode, you can configure your information in an automation script via command line mode.
+```bash
+The #set subcommand can set a certain configuration or configure multiple at the same time.
+Tccli configure set secretId AKIDwLw1234MMfPRle2g9nR2OTI787aBCDP
```
Comment (zqfan) [inline README.en.md]: When it is a command, you should not capitalize it, always be smaller case: Tccli -> tccli
The following commands all have same problem, please fix them, thanks

**Comment by zqfan** [comment]
Thanks for your help to translate the doc, it looks good, but there are some minor issues should be settled before it can be merged into master branch.

File: README.en.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+```bash
+[root@VM_180_248_centos ~]# tccli cvm DescribeZones
+{
+    "TotalCount": 4,
+    "ZoneSet": [
+        {
+            "ZoneState": "AVAILABLE",
+            "ZoneId": "100001",
+            "Zone": "ap-guangzhou-1",
+            "ZoneName": "Guangzhou District"
```
Comment (zqfan) [inline README.en.md]: ZoneName may not be translated because currently API response default setting is zh_CN

File: README.en.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+        },
+        {
+            "ZoneState": "AVAILABLE",
+            "ZoneId": "100002",
+            "Zone": "ap-guangzhou-2",
+            "ZoneName": "Guangzhou Second District"
```
Comment (zqfan) [inline README.en.md]: ditto

File: README.en.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+        },
+        {
+            "ZoneState": "AVAILABLE",
+            "ZoneId": "100003",
+            "Zone": "ap-guangzhou-3",
+            "ZoneName": "Guangzhou District III"
```
Comment (zqfan) [inline README.en.md]: ditto

File: README.en.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+        },
+        {
+            "ZoneState": "AVAILABLE",
+            "ZoneId": "100004",
+            "Zone": "ap-guangzhou-4",
+            "ZoneName": "Guangzhou Four Districts"
```
Comment (zqfan) [inline README.en.md]: ditto

File: README.en.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+4
+```
+3. Specify information for the Nth child of an array type object.
+```bash
+[root@VM_180_248_centos ~]# tccli cvm DescribeZones --filter ZoneSet[0]
+{
+    "ZoneState": "AVAILABLE",
+    "ZoneId": "100001",
+    "Zone": "ap-guangzhou-1",
+    "ZoneName": "Guangzhou District"
```
Comment (zqfan) [inline README.en.md]: ditto

**Comment by jaschadub** [comment]
Should be good now.

**Comment by zqfan** [comment]
Sorry for late feedback, but the international site has decide to has a separately package instead of reuse the Chinese version, please refer to https://github.com/tencentcloud/tencentcloud-cli-intl-en

**Pull request closed without merging**